cypress/integration/select-menu/position.feature

```diff
@@ -1,26 +1,23 @@
 Feature: Position of select menu dropdown
 
     Background:
-        # the default maximum
         Given the select menu dropdown has a height of 368px
-        # also the default maximum
         And the select menu dropdown has a width of 280px
-        And the select menu dropdown opens when the select is clicked
 
     Scenario: Default rendering
-        Given a select menu with more than 368px space below, before the end of the <body> element
-        When the select menu dropdown is rendered
+        Given the space below the select is more than its menu's height
```

Comment:
Again, I am unsure if 'space below' is specific enough because it doesn't define what that space is measured to.

Reply:
Done, see https://github.com/dhis2/ui-core/pull/527#issuecomment-562426939